bin/eslint_d.js

```diff
@@ -34,10 +34,17 @@ if (cmd === 'start') {
       var useStdIn = (process.argv.indexOf('--stdin') > -1);
 
       if (useStdIn) {
-        var concat = require('concat-stream');
-        process.stdin.pipe(concat({ encoding: 'string' }, function (text) {
+        var chunks = [];
+        process.stdin.setEncoding('utf8');
+
+        process.stdin.on('data', function (chunk) {
+          chunks.push(chunk.toString());
```

Comment:
And while we're talking nanoseconds already: http://jsperf.com/array-join-vs-string-connect
:wink: